metrics/application_metrics.go

```diff
@@ -0,0 +1,48 @@
+package metrics
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/konveyor/go-konveyor-tests/application"
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+var (
+	metricValue int
+	metricName  string = "konveyor_applications_inventoried"
+	apps        []api.Application
+	Application = application.Application
+)
+
+var _ = Describe("Application Metrics", Ordered, func() {
+	BeforeAll(func() {
+		// Get the current metric value
+		metricValue = GetMetricValue(metricName)
+	})
+
+	AfterAll(func() {
```

Comment:
I personally prefer to have AfterAll after the tests .

Reply:
I saw no examples of AfterAll at the end of test file, looks like this is the convention in Ginkgo